Incorrect / Invalid Brew Installation Instructions

After attempting to follow your installation guide, this isn't working for me:

``` shell
~ ❯❯❯ boot2docker up                                                          ⏎
Waiting for VM and Docker daemon to start...
.o
Started.
Writing /Users/kkirsche/.boot2docker/certs/boot2docker-vm/ca.pem
Writing /Users/kkirsche/.boot2docker/certs/boot2docker-vm/cert.pem
Writing /Users/kkirsche/.boot2docker/certs/boot2docker-vm/key.pem

To connect the Docker client to the Docker daemon, please set:
    export DOCKER_HOST=tcp://192.168.59.103:2376
    export DOCKER_CERT_PATH=/Users/kkirsche/.boot2docker/certs/boot2docker-vm
    export DOCKER_TLS_VERIFY=1

~ ❯❯❯ boot2docker shellinit
Writing /Users/kkirsche/.boot2docker/certs/boot2docker-vm/ca.pem
Writing /Users/kkirsche/.boot2docker/certs/boot2docker-vm/cert.pem
Writing /Users/kkirsche/.boot2docker/certs/boot2docker-vm/key.pem
    export DOCKER_HOST=tcp://192.168.59.103:2376
    export DOCKER_CERT_PATH=/Users/kkirsche/.boot2docker/certs/boot2docker-vm
    export DOCKER_TLS_VERIFY=1
~ ❯❯❯ brew install codeclimate
==> Installing codeclimate from codeclimate/homebrew-formulae
==> Downloading https://github.com/codeclimate/codeclimate/archive/v0.0.6.tar.gz
==> Downloading from: https://codeload.github.com/codeclimate/codeclimate/tar.gz
Already downloaded: /Library/Caches/Homebrew/codeclimate-0.0.6.tar.gz
==> env PATH=/usr/local/bin:/usr/local/Library/ENV/4.3:/usr/bin:/bin:/usr/sbin:/
bin/check
Get http:///var/run/docker.sock/v1.19/version: dial unix /var/run/docker.sock: no such file or directory. Are you trying to connect to a TLS-enabled daemon without TLS?
Unable to run `docker version', the docker daemon may not be running
Please ensure `boot2docker up' succeeds and you've run `boot2docker shellinit' in this shell
make: *** [install] Error 1

READ THIS: https://git.io/brew-troubleshooting
If reporting this issue please do so at (not Homebrew/homebrew):
  https://github.com/codeclimate/homebrew-formulae/issues
```

--- a/README.md
+++ b/README.md
@@ -2,6 +2,17 @@
 
 ## Usage
 
+Ensure docker is running or if you are using Boot2Docker, run:
+
+```console
+boot2docker up
+boot2docker shellinit
+```
+
+This will provide you with environment variables which should be set in your `.bash_profile` or `.zsh_profile` if they are not set. Otherwise, you will be told `Your environment variables are already set correctly.`
+
+Once you have those in your shell profile, run:
+
 ```console
 brew tap codeclimate/formulae
 brew install codeclimate  # for example

--- a/README.md
+++ b/README.md
@@ -6,10 +6,10 @@ Ensure docker is running or if you are using Boot2Docker, run:
 
 ```console
 boot2docker up
-boot2docker shellinit
+eval "$(boot2docker shellinit)"
 ```
 
-This will provide you with environment variables which should be set in your `.bash_profile` or `.zsh_profile` if they are not set. Otherwise, you will be told `Your environment variables are already set correctly.`
+This will set environment variables for your current session. To ensure these are loaded automatically in the future, these variables should be set in your `.bash_profile` or `.zsh_profile`. If these variables have already been added to your profile, you will be told `Your environment variables are already set correctly.`
 
 Once you have those in your shell profile, run:
 

